lib/country_code_picker.dart

```diff
@@ -163,11 +163,15 @@ class CountryCodePickerState extends State<CountryCodePicker> {
   }
 
   @override
-  void didUpdateWidget(CountryCodePicker oldWidget) {
-    super.didUpdateWidget(oldWidget);
+  void didChangeDependencies() {
+    super.didChangeDependencies();
 
     this.elements = elements.map((e) => e.localize(context)).toList();
-    _onInit(selectedItem);
```

Comment:
Where is `_onInit()` called from instead of this deleted line?

Reply:
Yes, that's an erro. In the last version I fixed it. Sorry.